_tsml_research_resources/soton/iridis/iridis_python.md

```diff
@@ -143,16 +145,22 @@ For CPU experiments start with one of the following scripts:
 
 >clustering_experiments.sh
 
+Within the above scripts you will need to adjust the top half section to work for your account. All the steps are described as comments within the script.
+
 You may need to use `dos2unix` to convert the line endings to unix format.
 
 The default queue for CPU jobs is _batch_.
 
+You can simply run `sh classification_experiments.sh` or alternativley write a slurm script to request the job. An example slurm script is found in this directory for running classification_experiments.sh.
+
+**NOTE: Ensure that anything you run has execution permissions with** `chmod +x SCRIPT_NAME`.
```

Comment:
Was this an issue?

Reply:
Yes I was getting permission errors, and I checked to see what file permissions were and saw that they weren't executable originally for me

Reply:
can keep it then, but reword i.e. "if you encounter error x, check y and run z"